### PR TITLE
Add required code change for Renown Level 62.

### LINKED ISSUE #33: Companion list UI reverts to original with > 20 companions
With Renown 62 and new companions from Torghast, it is possible to have 21 companions, and this causes left panel to revert the companion list from a Grid layout to the original layout when selecting companions for a mission.

Not sure if this is something that can be fixed through the global or has to be done in the original AddOn

### PR DESCRIPTION
**Pull request by FlipperPA**
Renown level 62 will give you your 21st follower, and VenturePlan is hard coded for a maximum of 20. This raises the max to 99 in a for loop.

Thanks for all of your work keeping this thing running - much appreciated.

EDIT: I wish I'd read the issues here before poking through the code, but... fixes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,16 @@ In the US, see e.g. [Galoob v. Nintendo](https://www.lexisnexis.com/community/ca
...
+To expose VenturePlan's internal data you will need to edit its source code. First, ensure that you are on the latest version, `4.16a`. Then open up `Interface/Addons/VenturePlan/vs.lua` in a text editor, and insert the line `_G[_] = T` in the first blank space. It should look like this when you're done:
 
 ![Notepad preview of changed file](img/notepad.png)
 
 This makes the internal data of the addon (`T`) available to other addons (by putting it in the global table, `_G`, which every addon has available to it). In general you shouldn't be messing with addons like this, because it's a great way of getting hacked, but in this case there's no way around it. _caveat emptor_
 
+# Hotfixing the Code for Renown Level 62 and Above
+
+At renown level 62, you will gain your 21st companion. VenturePlan is coded to handle a maximum to 20 companions. You will start seeing errors about `self.info` being `nil`. To fix this, change line 1960 (in version `4.16a`) from `for i=1,20 do` to `for i=1,99 do`. This will add support up to 99 companions.
```
Comment (cremor) [inline README.md:29]: You should mention the file name in which this needs to be changed.

**Comment by cremor** [comment]
As mentioned in https://github.com/hythloday/VenturePlanSoDMissions/issues/34#issuecomment-902449373 this is not a fix for #34

File: README.md (hunk 1 above)
Comment (mbinkhorst) [reply README.md:29]: it's in file: Widgets.lua

Comment (FlipperPA) [reply README.md]: Good catch, @cremor - I've updated the PR.

**Comment by hythloday** [comment]
Could you take a screenshot of the code after change in Notepad? Then I'll be happy to merge this.

Thanks for the work here.

**Comment by hythloday** [comment]
Oh yes, and please add yourself (and @cremor) if you would like, to the thanks section of the README, or I can do it post-merge.

**Comment by hythloday** [comment]
I admire your confidence in the technical ability of users! Let's see if it creates any confusion and I can change it if so.

**Pull request merged**